lib/legacy.coffee

```diff
@@ -1,3 +1,5 @@
+fs = require('fs')
```

Comment:
functions shouldn't use the name `path` because it conflicts with a node module. If there were a conflict here I'd rename the function's variable, not this one.